.github/workflows/build-gh-pages.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Build the book
       run: |
         rm -rf DISCOVER/_tags/*
-        jupyter-book build DISCOVER
+        sphinx-build -b html DISCOVER/ DISCOVER/_build/html
 
     - name: Push book HTML to gh-pages
       uses: peaceiris/actions-gh-pages@v3.9.0
```

Comment:
Add this option `destination_dir: dev/`, ref https://github.com/peaceiris/actions-gh-pages?tab=readme-ov-file#%EF%B8%8F-deploy-to-subdirectory-destination_dir